docs/source/parquet.mdx

```diff
@@ -163,4 +163,58 @@ Big datasets are partitioned into Parquet files (shards) of about 500MB each. Th
  "failed": []}
 ```
 
-To read and query the Parquet files, take a look at the [Query datasets from Datasets Server](parquet_process) guide.
\ No newline at end of file
+To read and query the Parquet files, take a look at the [Query datasets from Datasets Server](parquet_process) guide.
+
+## Using the Hugging Face Hub API
+
+For convenience, you can directly use the Hugging Face Hub `/api/parquet` endpoint which returns the list of Parquet URLs.
+
+It takes the dataset name, the configuration name and the split as parameters:
```

Comment:
we should mention that config and split are optional

Reply:
and maybe also showcase `https://huggingface.co/api/datasets/duorc/parquet`, since it's easy to type by hand (while adding the config and split names becomes a bit harder)